src/dom/index.js

```diff
@@ -58,10 +58,21 @@ export function setAccessor(node, name, old, value, isSvg) {
 		}
 		if (value && typeof value==='object') {
 			if (typeof old!=='string') {
-				for (let i in old) if (!(i in value)) node.style[i] = '';
+				for (let i in old) if (!(i in value)) {
+					if (~i.indexOf('-')) {
+						node.style.setProperty(i, '');
+					} else {
+						node.style[i] = '';
+					}
+				}
 			}
 			for (let i in value) {
-				node.style[i] = typeof value[i]==='number' && IS_NON_DIMENSIONAL.test(i)===false ? (value[i]+'px') : value[i];
+				if (~i.indexOf('-')) {
+					node.style.setProperty(i, value[i]);
+				}
+				else {
+					node.style[i] = typeof value[i]==='number' && IS_NON_DIMENSIONAL.test(i)===false ? (value[i]+'px') : value[i];
```

Comment:
Could you turn `IS_NON_DIMENSIONAL.test(i)===false` into `!IS_NON_DIMENSIONAL.test(i)`